devpage.html

```diff
@@ -75,7 +75,7 @@ <h3 class="mb-3">ScholarX</h3>
                             This free premium mentoring platform was created for Sri Lankans who see the opportunity and positive change brought about by a culture of knowledge and expertise sharing without the limitations of geographical borders."</p>
                             <div class="row">
                             
-                            <div class="col-md-5">
+                            <div class="col-md-5 pb-2">
```

Comment:
I think we can increase this a little bit more.

Reply:
@anjula-sack Increased it :)